src/ResourceManager/Compute/Commands.Compute/Extension/AEM/SetAzureRmVMAEMExtension.cs

```diff
@@ -61,11 +61,18 @@ public class SetAzureRmVMAEMExtension : VirtualMachineExtensionBaseCmdlet
                 Mandatory = false,
                 Position = 2,
                 ValueFromPipelineByPropertyName = false,
-                HelpMessage = "If this parameter is provided, the commandlet will not enable Windows Azure Diagnostics for this virtual machine.")]
+                HelpMessage = "Deprecated - Windows Azure Diagnostics is now disabled by default")]
         public SwitchParameter DisableWAD { get; set; }
 
         [Parameter(
                 Mandatory = false,
+                Position = 2,
+                ValueFromPipelineByPropertyName = false,
+                HelpMessage = "If this parameter is provided, the commandlet will enable Windows Azure Diagnostics for this virtual machine.")]
+        public SwitchParameter EnableWAD { get; set; }
```

Comment:
As far as I know, Parameter Sets need to contain at least one parameter specific to the set. How would I define parameter sets that only differ in those two switch parameters? Only using ResourceGroup and VMName must work. I also don't think this is a real issue. Using both switch parameters work